custom_import_map/public/plugin.tsx

```diff
@@ -20,6 +28,39 @@ export class CustomImportMapPlugin
     core: CoreSetup,
     { regionMap }: AppPluginSetupDependencies
   ): CustomImportMapPluginSetup {
+    // Register an application into the side navigation menu
+    core.application.register({
+      id: PLUGIN_ID,
```

Comment:
Here the id will determine URL path of Maps,
Example: http://localhost:5603/fol/app/customImportMap/create

We are using `maps-dashboards`, How about change id to `maps` or `maps-dashboards`?

Reply:
fixed.